variables.tf

```diff
@@ -44,6 +44,27 @@ variable "additional_public_subnets" {
   }))
 }
 
+variable "bastion_host_ami_filter" {
+  description = "List of maps used to create the AMI filter for the action runner AMI."
+  type        = map(list(string))
```

Comment:
What's the reason for using a `string` instead of `map(list(string))` ? And do we have to convert it back when exposing to the upstream module? I assume the module expects a `map(list(string))` instead of a `string`

Reply:
> What's the reason for using a string instead of map(list(string)) ?

The reason is both to improve the user-friendliness of our module and to keep in in-line with the rest of our modules, so that people know what to expect. `string` suffices for our use case, plus I think it's way more user-friendly to write

```
bastion_host_ami_name_filter = "ubuntu-*"
```

than

```
bastion_host_ami_name_filter = {
  "whatever" : [
    "ubuntu-*"
  ]
}
```

don't you agree? 😉

> And do we have to convert it back when exposing to the upstream module?

Yes, that's why I mentioned 

> some Terraform manipulation

Reply:
Makes sense.